src/SourceBuild/content/eng/tools/tasks/Microsoft.DotNet.SourceBuild.Tasks.LeakDetection/DummyAttributeTypeProvider.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Reflection;
+using System.Reflection.Metadata;
+using System.Reflection.Metadata.Ecma335;
+
+namespace Microsoft.DotNet.SourceBuild.Tasks.LeakDetection
+{
+    internal class DummyAttributeTypeProvider : ICustomAttributeTypeProvider<Type>
```

Comment:
Consider including a brief comment indicating why an empty/dummy implementation exists
```suggestion
    // An empty ICustomAttributeTypeProvider implementation necessary to read metadata attribute values.
    internal class DummyAttributeTypeProvider : ICustomAttributeTypeProvider<Type>
```